modelkit/core/model.py

```diff
@@ -392,15 +388,18 @@ def _validate(
     ):
         if model:
             try:
-                if self.service_settings.enable_validation:
```

Comment:
Breaking change: dropped the `enable_validation` / `MODELKIT_ENABLE_VALIDATION` flag.
This has worked (with `pydantic < 2`)  for rather simple `pydantic` `Models`, but not complex ones with nested structures. (see: https://github.com/Cornerstone-OnDemand/modelkit/pull/8) 
It still is an open question in pydantic 2, whether to allow recursive construction of models without validation (see: https://github.com/pydantic/pydantic/issues/8084).
What do you guys think ?

Reply:
yes it goes perfectly with pydantic moves to remove stuff that they never managed to make bullet-proof.
I vote to remove it until there is a bullet-proof solution to offer.

Reply:
I am also tempted to keep this flag, while logging a deprecation warning saying it now does nothing (hence leading to no breaking changes for the migration!)